lineoperations/README

```diff
@@ -26,23 +26,72 @@ Features
 Usage
 =====
 
-After the plugins has been installed successfully, load the plugin via 
-Geany's plugin manager and a new menu item in the Tools menu will 
-appear. Click on each menu item to apply the operation on whole file, or 
-selection. See descriptions below to see operations for each menu item.
+After the plugins has been installed successfully, load the plugin via
+Geany's plugin manager and a new menu item in the Tools menu will
+appear ("Tools" > "Line Operations"). Click on each menu item to apply
+the operation on whole file, or selection. See descriptions below to
+see operations for each menu item.
+
+Preferences
+-----------
+
+To enable preferences ("Tools" > "Plugin Manager"), check the checkbox to
+enable Line Operations, and with the item highlighted click "Preferences"
+on bottom of the "Plugin Manager" frame.
+
+**Use collation based string compare** - When this option is enabled it will
+compare strings using linguistically correct rules for the current locale
+(g_utf8_collate).
+
+When it is disabled it will compare strings based off numerical value of the
```

Comment:
Should it be _"based off"_ or _"based on"_?

Reply:
I think they're both [valid](https://english.stackexchange.com/questions/15081/based-on-instead-of-based-off-of). I changed it to "based on", I think it sounds a little better.